docs/workflow/requirements/linux-requirements.md

```diff
@@ -62,7 +62,7 @@ Minimum RAM required to build is 1GB. The build is known to fail on 512 MB VMs (
 Toolchain Setup
 ---------------
 
-Building the repo requires CMake 3.14.2 or newer on Linux. Add Kitware's APT feed to your configuration for a newer version of CMake. See their instructions at <https://apt.kitware.com/>. Also, add LLVM/s APT feed to your configuration for a newer version of CMake. See their instructions as <http://apt.llvm.org/>.
+Building the repo requires CMake 3.14.2 or newer on Linux. Add Kitware's APT feed to your configuration for a newer version of CMake. See their instructions at <https://apt.kitware.com/>. Also, add LLVM's APT feed to your configuration for a newer version of clang/LLVM. See their instructions at <https://apt.llvm.org/>.
```

Comment:
Maybe we should say "you may need to add LLLVM's APT feed ....".  E.g. even Ubuntu 18.04 already has clang-9 package.